commercetools.NET/CustomObjects/CustomObject.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Newtonsoft.Json;
+
+namespace commercetools.CustomObjects
+{
+    /// <summary>
+    /// Custom objects are a way to store arbitrary JSON-formatted data on the commercetools platform.
+    /// <see href="https://docs.commercetools.com/http-api-projects-custom-objects.html#customobject"/>
+    /// </summary>
+    public class CustomObject
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; private set; }
+        
+        [JsonProperty(PropertyName = "version")]
+        public int Version { get; private set; }
+        
+        [JsonProperty(PropertyName = "container")]
+        public string Container { get; private set; }
+        
+        [JsonProperty(PropertyName = "key")]
+        public string Key { get; private set; }
+        
+        [JsonProperty(PropertyName = "createdAt")]
+        public DateTime CreatedAt { get; private set; }
+        
+        [JsonProperty(PropertyName = "lastModifiedAt")]
+        public DateTime LastModifiedAt { get; private set; }
+        
+        [JsonProperty(PropertyName = "value")]
+        public object Value { get; private set; }
```

Comment:
I'm not sure about which type to use here. CommerceTools supports any JSON value (number, string, array, object, etc), so we might be able to use `JToken` instead of `object`.

Reply:
Alternatively, we could add a type parameter `T`, which makes it a lot easier for consumers.